apps/settings/js/panels/display/slider_handler.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Update setting value based on slider position.
+ * Base on sound panel's implementation.
+ *
+ * @module SliderHandler
+ */
+define(function(require) {
+  'use strict';
+
+  const INTERVAL = 300;
+
+  var SliderHandler = function() {
+    this._element = null;
+    this._key = '';
+    this._isFirstInput = false;
+    this._intervalID = null;
+  };
+
+  SliderHandler.prototype = {
+    /**
+     * initialization
+     *
+     * The sliders listen to input, touchstart and touchend events to fit
+     * the ux requirements, and when the user tap or drag the sliders, the
+     * sequence of the events is:
+     * touchstart -> input -> input(more if dragging) -> touchend -> input
+     *
+     * @access public
+     * @memberOf SliderHandler.prototype
+     * @param  {Object} element html elements
+     * @param  {String} settings key
+     */
+    init: function d_init(element, key) {
+      this._element = element;
+      this._key = key;
+
+      this._boundSetSliderValue = function(value) {
+        this._setSliderValue(value);
+      }.bind(this);
+
+      this._element.addEventListener('touchstart',
+        this._touchStartHandler.bind(this));
+      this._element.addEventListener('input',
+        this._inputHandler.bind(this));
+      this._element.addEventListener('touchend',
+        this._touchEndHandler.bind(this));
+    },
+
+    /**
+     * Change slider's value
+     *
+     * @access private
+     * @memberOf SliderHandler.prototype
+     * @param {Number} value slider value
+     */
+    _setSliderValue: function d_setSliderValue(value) {
+      this._element.value = value;
+    },
+
+    /**
+     * Handle touchstart event
```

Comment:
It will be nice to describe that change make the slider not being moved when scrolling through the slider